src/App.tsx

```diff
@@ -33,19 +33,41 @@ function App() {
     const [configStr, setConfigStr] = useState<string>("");
     const [viewRecipe, setViewRecipe] = useState<boolean>(true);
     const [viewConfig, setViewConfig] = useState<boolean>(true);
+    const [viewResults, setViewResults] = useState<boolean>(false);
     const [viewLogs, setViewLogs] = useState<boolean>(true);
+    const [runTime, setRunTime] = useState<number>(0);
 
-    const submitRecipe = async () => {
+    let start = 0;
+
+    async function sleep(ms: number): Promise<void> {
+        return new Promise((resolve) => setTimeout(resolve, ms));
+    };
+
+    const submitRecipe = async (useECS: boolean = false) => {
+        setResultUrl("");
         const firebaseRecipe = "firebase:recipes/" + selectedRecipe
         const firebaseConfig = "firebase:configs/" + selectedConfig;
-        const url = getSubmitPackingUrl(firebaseRecipe, firebaseConfig);
+        const url = getSubmitPackingUrl(firebaseRecipe, firebaseConfig, useECS);
         const request: RequestInfo = new Request(url, {
-            method: "POST",
+            method: useECS ? "GET" : "POST",
         });
+        start = Date.now();
         const response = await fetch(request);
         const data = await response.json();
-        setJobId(data.jobId);
-        return data.jobId;
+        if (useECS) {
+            if (response.ok) {
+                const range = (Date.now() - start) / 1000;
+                setRunTime(range);
+                setJobId(data.job_id);
```

Comment:
Should we fix snake_case for consistency? `job_id` vs `jobId`

Reply:
good point! I need to make this change in the cellpack server.py, because that's where I define the response type, but I think camel case `jobId` is better